Validation/RecoTrack/interface/MultiTrackValidator.h

```diff
@@ -104,10 +106,14 @@ class MultiTrackValidator : public DQMGlobalEDAnalyzer<MultiTrackValidatorHistog
                                 std::vector<size_t>& selected_tPCeff) const;
   size_t tpDR(const TrackingParticleRefVector& tPCeff,
               const std::vector<size_t>& selected_tPCeff,
-              DynArray<float>& dR_tPCeff) const;
+              DynArray<float>& dR_tPCeff,
+              DynArray<float>& dR_tPCeff_jet,
+              const edm::View<reco::Candidate>* cores) const;
   void trackDR(const edm::View<reco::Track>& trackCollection,
-               const edm::View<reco::Track>& trackCollectionDr,
-               DynArray<float>& dR_trk) const;
+                                    const edm::View<reco::Track>& trackCollectionDr,
+                                    DynArray<float>& dR_trk,
+                                    DynArray<float>& dR_trk_jet,
+                                    const edm::View<reco::Candidate>* cores) const;
```

Comment:
Indentation is off.

Reply:
done. sorry for all these indentation problems: my editor changed it by default